rholang/src/test/scala/coop/rchain/rholang/interpreter/ReduceSpec.scala

```diff
@@ -28,14 +28,10 @@ import scala.collection.mutable.HashMap
 import scala.concurrent.Await
 import scala.concurrent.duration._
 
+final case class TestFixture(space: RhoISpace, reducer: Reduce[Task])
+
 trait PersistentStoreTester {
-  def withTestSpace[R](
-      f: FreudianSpace[Channel,
-                       BindPattern,
-                       OutOfPhlogistonsError.type,
-                       ListChannelWithRandom,
-                       ListChannelWithRandom,
-                       TaggedContinuation] => R): R = {
+  def withTestSpace[R](errorLog: ErrorLog)(f: TestFixture => R): R = {
```

Comment:
Would prefer just
`def withTestFixture[R](errorLog: ErrorLog)(f: (Rspace, Reducer) => R): R = {`
but that's mainly FYI - the duplication has been reduced :ok_hand: